Makefile

```diff
@@ -204,3 +204,7 @@ upload-packages:
 .PHONY: vendor-update
 vendor-update:
 	glide update --strip-vendor
+
+.PHONY: openshiftci-presubmit-e2e
+openshiftci-presubmit:
```

Comment:
or this can be `openshiftci-presubmit-e2e`

Reply:
ah, yes, that is my mistake